INSTALL.md

```diff
@@ -20,17 +21,30 @@ If you do not want to grant the ruby package manager, [RubyGems](https://rubygem
 To install CocoaPods you can grab the right version by using `bundler` (recommended) or you can directly install it with RubyGems:
 
 ```
-gem install cocoapods
+$ gem install cocoapods
 ```
 
 In the last case please ensure that you are using the same version as indicated at the end of the `Podfile.lock` file.
 
+### Install XcodeGen
+
+You can directly install XcodeGen with [Homebrew](https://brew.sh) or RubyGems:
+
+```
+$ brew install xcodegen
+```
+or
+
+```
+$ gem install xcodegen
```

Comment:
I failed to install using this command. XcodeGen promotes mint instead of gems. Can we advice users to use only HomeBrew for the moment.

Note that Mint is the only to manage different version of XcodeGen (https://github.com/yonaskolb/XcodeGen/issues/757#issuecomment-576761905)